e2e/vm-sidebar.e2e-spec.ts

```diff
@@ -0,0 +1,234 @@
+    sidebar.clickColorChange().then(color => {
+      sidebar.clickClose();
+      vmlist.getVMColor(0).then(vmcolor => {
+        expect(vmcolor).toContain(color);
+      });
+    });
+  });
+
+  it('Verify description is changed', () => {
+    vmlist.clickOpenSidebar(0);
+    sidebar.setDescription('desc');
+    sidebar.clickClose();
+    vmlist.clickBell();
+    expect(vmlist.verifyBellMessage('Description changed')).toBeTruthy();
+  });
+
+  it('Verify new group is set', () => {
+    vmlist.clickOpenSidebar(0);
+    sidebar.clickEditGroup();
+    sidebar.setNewGroupOption(sidebar.group);
+    sidebar.clickSubmitGroupButton();
+    sidebar.waitGroupChanged(sidebar.group);
+    sidebar.clickClose();
+    vmlist.clickBell();
+    expect(vmlist.verifyBellMessage('Instance group changed')).toBeTruthy();
+  });
+
+  it('Verify existing group can be set', () => {
+    vmlist.clickOpenSidebar(0);
+    sidebar.clickEditGroup();
+    sidebar.setExistingGroupOption().then(group => {
+      sidebar.clickSubmitGroupButton();
+      sidebar.waitGroupChanged(group);
+      expect(sidebar.getGroup()).toEqual(group);
+    });
+    sidebar.clickClose();
+    vmlist.clickBell();
+    expect(vmlist.verifyBellMessage('Instance group changed')).toBeTruthy();
+  });
+
+  it('Verify group with incorrect name can not be set', () => {
+    vmlist.clickOpenSidebar(0);
+    sidebar.clickEditGroup();
+    sidebar.setNewGroupOption('5214351538713');
+    expect(sidebar.isEnabledSubmitGroupButton()).toBeFalsy(
+      'Create button is enabled for incorrect group name',
+    );
+    sidebar.clickCancelGroupButton();
+    browser.wait(
+      protractor.ExpectedConditions.visibilityOf(element(by.css(`.open`))),
+      2000,
+      'No sidebar',
+    );
+    sidebar.clickClose();
+  });
+
+  it('Verify group is removed', () => {
+    vmlist.clickOpenSidebar(0);
+    sidebar.clickEditGroup();
+    sidebar.setRemoveGroupOption();
+    sidebar.clickSubmitGroupButton();
+    sidebar.waitGroupChanged('Default group');
+    sidebar.clickClose();
+    vmlist.clickBell();
+    expect(vmlist.verifyBellMessage('Instance group removed')).toBeTruthy('No bell message');
+  });
+
+  it('Change SO to fixed SO for running VM', () => {
+    vmlist.clickOpenSidebarRunning();
+    sidebar.clickChangeSO();
+    sidebar.selectFixedTab();
+    sidebar.changeSO();
+    expect(sidebar.checkWarning()).toEqual('Virtual machine will be restarted');
+    const temp = sidebar.getSelectedSO();
+    sidebar.clickYesDialogButton();
+
+    sidebar.isEnabledChangeSO();
+    sidebar.isEnabledAddAffinityGroup();
+    sidebar.isEnabledAddSSHKey();
+    vmlist.clickBell();
+    expect(vmlist.verifyBellMessage('VM stopped')).toBeTruthy();
+    expect(vmlist.verifyBellMessage('Service offering changed')).toBeTruthy();
+    expect(vmlist.verifyBellMessage('VM started')).toBeTruthy();
+    expect(sidebar.getChangedSOName()).toEqual(temp);
+    sidebar.clickClose();
+  });
+
+  // custom SO for stopped VM
```

Comment:
Did you forget to write this test or is there another reason for this comment? Remove it if it is not needed.